packages/tezos-rust-libs/tezos-rust-libs.1.4/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Tezos: all rust dependencies and their dependencies"
+maintainer: "contact@tezos.com"
+authors: "Tezos devteam"
+license: "multiple"
```

Comment:
```suggestion
license: "LicenseRef-multiple"
```
is the escape hatch for this kind of thing in the SPDX syntax

Reply:
Thanks. [Upstreaming it](https://gitlab.com/tezos/tezos-rust-libs/-/commit/0b86fd68a92b3aa9281058ed5fb8886d430a8bf9)